data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+		payment 10000000
+		conversation
+			`With a strong force ensuring that no one can interrupt the final construction stage, space-based cranes set about removing the three outer sections of the Geocoris and moving them to the steel ring deployed around the original station core. Within a few hours all seven of the new Geocoris have been dismantled and integrated to the station structure, using lessons learnt from the construction of Mountaintop to rapidly finish a station that is fully battle-ready, if lacking in some of the amenities of life. In a reversal of the Mountaintop priorities, the construction workers unpack and assemble a massive holoprojector while wearing environmental suits before they begin unpacking the contents of the Geocoris storage pods and engaging principle life support systems.`
+			`	As soon as the life support systems around the docking section are engaged, the security ships begin landing those damaged by the hastily assembled pirate response in order to affect critical repairs.`
+			`	As you're attending to your own ships after their extended guard duty, you get pinged by a hail that has been relayed through from Mountaintop. When you answer it you are greeted by Xilin and Hiyamaharu who look very pleased.`
+			`	"We hear that the operation to establish Mooncreek has been successful Captain <last>! How'd things go with you?"`
+			choice
+				`	"Yeah, it was fairly chill."`
+					goto fantastic
+				`	"It was very close out there, there were some losses."`
+			`	Hiyamaharu is inscrutable but Xilin adopts a sombre expression, "Unfortunately there was always a risk of that in this endeavor."`
+			`	"Still," says Hiyamaharu.`
+			label fantastic
+			`	"A great success all round," says Xilin.`
+			`	"The Republic and Hai governments have approved a reward of <payment> by the way, as thanks for your considerable assistance here."`
+			`	"If you have a moment though, we'd appreciate your continued assistance with a bit of PR now. To finalize the station the plan is to facilitate some of the major outfitters to set up on the new station, both from Hai space and from ours. Predominantly this would be so that we can ensure a ready supply of munitions on the station at all times. As well, the long-term garrison to act as security needs to be established. We'd like you to escort each of them on the last leg of their journeys, mostly for the photo ops."`
+			`	"Of course," says Xilin, "we'll understand if you're too busy now."`
+			choice
+				`	"Sure, why not?"`
+					goto escort
+				`	"I think I've had enough of the limelight for now."`
+			`	Xilin performs a combination shrug and half-nod, "Fair enough <last>, be safe out there. We'll send you an invite for the final treaty ceremony. We'd all like you to be there."`
+			`	The call ends, leaving you to your own devices.`
+				decline
+			label escort
+			`	"Lovely, in that case the first convoy will be waiting for you on Maelstrom. I'll meet you on Mooncreek when you return."`
```

Comment:
```suggestion
			`	"Lovely. In that case, the first convoy will be waiting for you on <destination>. I'll meet you there on <origin> when you return."`
```

Reply:
I appreciate all the time you catch my just complete failure to remember that text replacements are a thing.